client/data/authorizations/hooks.ts

```diff
@@ -14,34 +14,57 @@ import {
 	Authorizations,
 } from 'wcpay/types/authorizations';
 
-export const useAuthorizations = ( {}: Query ): Authorizations =>
-	useSelect( ( select ) => {
-		const {
-			getAuthorizations,
-			getAuthorizationsError,
-			isResolving,
-		} = select( STORE_NAME );
+export const useAuthorizations = ( {
+	paged,
+	// eslint-disable-next-line @typescript-eslint/naming-convention
+	per_page,
```

Comment:
I'd suggest to use an [alias](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) to make eslint happy.
```suggestion
	per_page: perPage,
```

Reply:
The reason why I kept this and used the `eslint-disable` instead is mainly the typing.

The `Query` type (`client/transactions/declarations.d.ts`) has a `per_page` parameter, and returning `perPage` will raise typing errors  in many places that needs to be also silenced/worked on. I think fixing typing errors and introducing workarounds is not worth the effort compared to a one `eslint-disable` here.

What do you think?

Reply:
It's okay to stick with the `eslint-disable`, but please note that using  an alias (`per_page: perPage`) allows you to treat the var as perPage within the scope of the function without modifying it to 'outside world'. What I am trying to say is that you can work with the perPage alias within the function, making eslint happy, while in the TS type it remains as per_page, or whatever you need.

Reply:
I only raised the TS issue because I've seen your suggestion to change the `per_page` in the `query` object too. Which will raise an error given it will be different than the `Query` type.

```js
// This will raise errors as the query object should be casted to Query type
perPage: Number.isNaN( parseInt( per_page ?? '', 10 ) )
```

But I agree with you that changing it inside the function only should be fine.